### ISSUE TITLE
combat messages log

### ISSUE THREAD

**Issue by Grokmoo** [Grokmoo/sulis#121]
Add a combat log or messages log that is optional but probably on by default.

This will explain the meaning of the combat feedback for new players.

**Comment by SavainB** [comment]
can i work on this ?

**Comment by Grokmoo** [comment]
Yes, this would definitely be something I would accept a PR for.
